samples/bluetooth/hci_uart/prj.conf

```diff
@@ -8,16 +8,18 @@ CONFIG_BT=y
 CONFIG_BT_HCI_RAW=y
 CONFIG_BT_HCI_RAW_H4=y
 CONFIG_BT_HCI_RAW_H4_ENABLE=y
-CONFIG_BT_BUF_ACL_RX_SIZE=255
-CONFIG_BT_BUF_CMD_TX_SIZE=255
-CONFIG_BT_BUF_EVT_DISCARDABLE_SIZE=255
-CONFIG_BT_CTLR_ASSERT_HANDLER=y
 CONFIG_BT_MAX_CONN=16
 CONFIG_BT_TINYCRYPT_ECC=n
-CONFIG_BT_CTLR_DTM_HCI=y
 
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
 
 # Workaround: Unable to allocate command buffer when using K_NO_WAIT since
 # Host number of completed commands does not follow normal flow control.
 CONFIG_BT_BUF_CMD_TX_COUNT=10
+
+# CONFIG_BT_BUF_EVT_RX_COUNT=30
+
+# CONFIG_BT_BUF_EVT_RX_SIZE=255
+# CONFIG_BT_BUF_ACL_RX_SIZE=255
+# CONFIG_BT_BUF_ACL_TX_SIZE=251
+# CONFIG_BT_BUF_CMD_TX_SIZE=255
```

Comment:
Are these intentionally left commented-out?

Reply:
Added comments in the file.